app/assets/stylesheets/sephora_style_guide/partials/_slider.scss

```diff
@@ -0,0 +1,145 @@
+.range-slider {
+  padding-left: 12px;
+  padding-right: 12px;
+}
+
+.irs {
```

Comment:
These styles look 3rd party. Could we put them in another file so it's easy to upgrade? If you want to keep it clean, you should then at least adopt them to follow our code style (like removing vendor prefixes...).